src/go/k8s/pkg/console/store.go

```diff
@@ -178,6 +203,12 @@ func (s *Store) getKafkaClientCertKey(
 	return fmt.Sprintf("%s-%s-%s", cluster.GetNamespace(), cluster.GetName(), kafkaClientCertSuffix)
 }
 
+func (s *Store) getAdminAPIClientCertKey(
+	cluster *redpandav1alpha1.Cluster,
+) string {
+	return fmt.Sprintf("%s-%s-%s", cluster.GetNamespace(), cluster.GetName(), adminAPIClientCertSuffix)
+}
```

Comment:
NIT: This function should be introduced in previous commit.

Reply:
Updated